nix/pkgs/libspdk/default.nix

```diff
@@ -61,8 +61,8 @@ let
     src = fetchFromGitHub {
       owner = "openebs";
       repo = "spdk";
-      rev = "4b3b937d1f0903d41c1252459b7578a2fcb00bee";
-      sha256 = "sha256-tCc+xFO9w9GmmwfxI8xp8Zs/QI1DqQ7voSGSp3K3Cic=";
+      rev = "d2e25457f879c7d228793f7204a49fedbfc2eca6";
```

Comment:
Version parameters change is needed ?
version = "23.01-4b3b937"  => version = "23.01-d2e2545";

Reply:
Good point. Yes, this needs to be updated as well.